### PR TITLE
Add schema based on diagram sql

### PR DESCRIPTION
**Pull request by mumo-esther**
- Create Clinic Database.
- Create patients, medical-histories, invoices, invoice-items and treatment tables schemas based on the diagram provided.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    invoice_id INTEGER NOT NULL,
+    treatment_id INTEGER NOT NULL
+);
+ALTER TABLE
+    invoice_items ADD PRIMARY KEY(id);
+CREATE TABLE treatments(
+    id INTEGER NOT NULL,
+    type VARCHAR(255) NOT NULL,
+    name VARCHAR(255) NOT NULL
+);
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql]: - [ ] Kindly implement all "join tables" for the many-to-many relationshipbetween the `medical_histories` and `treatments` tables. You can create a new table like the following:

```sql
  CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ALTER TABLE
+    invoices ADD CONSTRAINT invoices_medical_history_id_foreign FOREIGN KEY(medical_history_id) REFERENCES medical_histories(id);
+ALTER TABLE
+    invoice_items ADD CONSTRAINT invoice_items_treatment_id_foreign FOREIGN KEY(treatment_id) REFERENCES treatments(id);
+ALTER TABLE
+    invoice_items ADD CONSTRAINT invoice_items_invoice_id_foreign FOREIGN KEY(invoice_id) REFERENCES invoices(id);
+ALTER TABLE
+    treatments ADD CONSTRAINT treatments_id_foreign FOREIGN KEY(id) REFERENCES medical_histories(id);
+ALTER TABLE
+    medical_histories ADD CONSTRAINT medical_histories_patient_id_foreign FOREIGN KEY(patient_id) REFERENCES patients(id);
```
Comment (ShadyShawkat) [inline schema_based_on_diagram.sql]: - [ ] Please create indexes for each of the created foreign keys.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @mumo-esther, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — APPROVED** [review]
Hi @mumo-esther 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Correct use of GitFlow ✔️
Descriptive commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**